### ISSUE TITLE
[Windows/Update 1.8.1 ==> 2.0.0] Logfiles in NCPA root install dir

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#283]
Hi!

After upgrade there are both logfiles (ncpa_listener.log, ncpa_passive.log) available in the new location: ```NCPA_INSTALL_PATH/var/```:
```
22:15:41 [user@win_7_enterprise_german]:/cygdrive/c/Program Files (x86)/Nagios/NCPA$ ls -lah ./var/*.log
-rwxr-x---+ 1 Administratoren Domain Users 151K 16. Dez 22:15 ./var/ncpa_listener.log
-rwxrwx---+ 1 Administratoren SYSTEM       2,4M 16. Dez 22:15 ./var/ncpa_passive.log
```

Additionally they were created in the ```NCPA_INSTALL_PATH```:
```
22:15:12 [user@win_7_enterprise_german]:/cygdrive/c/Program Files (x86)/Nagios/NCPA$ ls -lah ./*.log
-rwxrwx---+ 1 Administratoren SYSTEM 65 16. Dez 22:15 ./ncpa_listener.log
-rwxrwx---+ 1 Administratoren SYSTEM 65 16. Dez 22:15 ./ncpa_passive.log
```

But it seems, that they were only created at service start and they contains only this line:
```
22:15:59 [user@win_7_enterprise_german]:/cygdrive/c/Program Files (x86)/Nagios/NCPA$ cat ncpa_passive.log
[08348] 2016/12/16 22:15:27.110 starting logging at level ERROR

22:16:00 [user@win_7_enterprise_german]:/cygdrive/c/Program Files (x86)/Nagios/NCPA$ cat ncpa_listener.log
[07720] 2016/12/16 22:15:09.348 starting logging at level ERROR
```

**Comment by jomann09** [comment]
I believe old versions of NCPA placed log files by default in `INSTALL_PATH/var` but now we place them by default in `INSTALL_PATH/var/log` on new installs (since it now properly figures out your set location in ncpa.cfg.

The second set of logs that is directly in `INSTALL_PATH` is actually separate logging for the windows services themselves - if something stops Python from starting the logs will never get populated in `var` so those are the only way to find out what is wrong. If you look at your logs in the main directory on that Windows XP system you'll see what I mean. The logging happens inside the `win32service` base that  comes with cx_Freeze.

Considering all this, I will take a look at the code/options and see if I can't rename them to something else so that they aren't confusing.

**Comment by ccztux** [comment]
This would be great. Thank you.

**Comment by jomann09** [comment]
Fixed this with caf81c76cbd2e540ffea64b5cafbe3ab3baabcf2
